mappings/net/minecraft/nbt/NbtByte.mapping

```diff
@@ -1,17 +1,27 @@
 CLASS net/minecraft/class_2481 net/minecraft/nbt/NbtByte
-	COMMENT Represents an NBT byte.
+	COMMENT Represents an NBT byte. Its type is {@value NbtElement#BYTE_TYPE}.
```

Comment:
Maybe document that if they are immutable or not, as this is the case for some number tags but not for list or compounds.